packages/verification-pool-api/deployment/deploy.ts

```diff
@@ -1,10 +1,19 @@
 // Script used for deploying to firebase
 // To use call ts-node deploy.ts --celo-env YOUR_ENV_NAME --celo-proj YOUR_PROJECT_NAME
 
-import * as util from '@celo/verification-pool-api/deployment/deployment-utils'
-import * as parseArgs from 'minimist'
+import * as util from './deployment-utils'
+import parseArgs from 'minimist'
 
-const envArg = parseArgs(process.argv.slice(2))['celo-env']
-util.setProject(parseArgs(process.argv.slice(2))['celo-proj'])
-util.setEnv(envArg)
-util.deploy(envArg)
+async function main() {
+  const envArg = parseArgs(process.argv.slice(2))['celo-env']
+  const project = parseArgs(process.argv.slice(2))['celo-proj'] || 'celo-testnet'
+  await util.setProject(project)
+  await util.setEnv(envArg)
+  try {
+    await util.deploy(envArg)
+  } catch (e) {
+    console.log(e)
```

Comment:
Nit: this should be `console.error` and it's best to add a useful error message first. E.g.
`console.error('Verification pool deployment failed', e)`